app/_assets/stylesheets/footer.less

```
@@ -336,70 +297,19 @@
    */
 
   .logo-wrapper {
-    .flex-col(1, 1, 12);
     margin: 0.5rem 0 @gutter-flex-1x 0;
-
-    @media (min-width: @grid-width-lg) {
-      .flex-col(1, 1, 2);
-    }
   }
-  .image,
   .logo {
-    width: 100%;
     max-width: 120px;
-    height: 100%;
 
     a {
       display: block;
     }
     img {
       width: 100%;
```

```suggestion
      width: 90%;
```
Maybe? Setting it to 90% makes it match the .com, but I don't know if it's bad practice to do this with an image.